### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by SamanAtashi**
# Math magicians: full website
Features introduced are as follows:
- React routers
- styling components using basic styling
- Added Pages : `Home` , `Calculator` , `Quote`

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
 
-const App = () => (<Calculator />);
+const App = () => (
+  <>
+    <Navbar />
+    <Switch>
+      <Route path="/calculator">
+        <div className="App">
+          <h2>Let&apos;s do some Calculation!</h2>
+          <Calculator />
```
Comment (arslanbisharat) [inline src/App.js:16]: - [ ] The Calculator is not working properly. Please have a look at the Gif below to know what is happening.

File: src/index.css [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-  position: relative;
+*::selection {
+  background-color: tomato;
 }
 
 body {
   width: 100vw;
   height: 100vh;
 }
+
```
Comment (arslanbisharat) [inline src/index.css:16]: - [ ] [Optional] Let's add a live demo link in your Readme.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mke2111 — APPROVED** [review]
### Hi @SamanAtashi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**